app/assets/stylesheets/posts.scss

```diff
@@ -1,3 +1,61 @@
 // Place all the styles related to the posts controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+body {
+  // image-url is coming from rails
+  // http://guides.rubyonrails.org/asset_pipeline.html
+  background: image-url("indian_arm.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+
+.main-content{
+  background: rgba(157, 159, 164, .5);
+  border-radius: 10px;
+  // background-color: #9D9FA4;
+}
+
+textarea.blog { height: 200px;}
+
+h1 {
+  color: #fcf3e7;
+  font-family: fantasy;
+  font-weight: 600;
+  // text-shadow: 2px 2px 6px #000000;
+  text-shadow: -2px 0 black, 0 2px black, 2px 0 black, 0 -2px black;
+}
+
+.well {
+  background-color: rgba(157, 159, 164, .9);
+
+}
+
+a {
+  color: #141415;
+  font-family: sans-serif;
+  font-weight: bold;
+}
+a:hover {
+  color: #CCCCCC;
+}
+a:active {
+  color: #333333;
+}
+.post-thumb {
+  width: 200px;
+  height: 200px;
+  background-color: rgba(157, 159, 164, .5);
+}
+.big-icon {
+  font-size: 30px;
+  color: #A7232E;
+}
+
+p {
+  font-family: cursive;
+  font-size: 16px;
+  color: black;
+}
```

Comment:
make sure you separate the the selectors with spaces, your missing line spaces on line 46 and 52